### ISSUE TITLE
add ability for vak.core.predict to save outputs of trained model

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#90]

**Comment by NickleDave** [comment]
e.g. command-line option that does predict but doesn't apply argmax to softmax layer

**Comment by NickleDave** [comment]
would be nice to get this per-syllable too instead of just vectors of raw probabilities for each time bin, as suggested by Archana Nagarajan. Could give mean probability per syllable in generated csv of predictions?

**Comment by NickleDave** [comment]
thinking more about this

@JaerongA kindly provided me a modified version of `vak.core.predict` he created to save logits -- after seeing that, and discussing with other people that would like similar functionality, I am thinking most general approach will be:

- [ ] add an option in the `[PREDICT]` section of the config.toml file called `save_logits` which can be `true` or `false`.
- [ ] add similar `save_logits` bool flag to `vak.core.predict` function with default `False`
- [ ] within `vak.core.predict`, check if `save_logits` is `True`, and if so, for each `spect_path`, save another file (`logits_path`) in `results_dir` that contains a matrix of shape (n classes x t time bins). The `vak.core.predict` function will be responsible for reshaping the output from the network to have this shape, by removing the batch dimension and any padding added.

I am hesitant to add any code that does any further manipulation of the outputs. I'd rather save the "raw" outputs to make it easier for a user to get access to them, and then leave it up to the user to transform those in whatever way suits their needs.
